baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -113,7 +113,7 @@ For example, if all of your data needs to be sent to a single node, you could pr
 baggage: serverNode=DF:28
 ```
 
-For example, if you need to log the original user ID when making transactions arbitrarily deep into a trace.
+For example, if you need to log the original user ID when making transactions arbitrarily deep into a distributed request.
```

Comment:
Sentence doesn't really work. The overall example is about annotating logs with request-specific information.

Reply:
User-id could be considered request-specific information

Reply:
Thanks, I have reworded this statement now to attempt to improve this.